### PR TITLE
Update reporting.md

### PR DESCRIPTION
**Pull request by ighristov**
Added info for Blazor Native Report Viewer, changed a bit the existing paragraph, explaining the old Blazor Viewer internally uses HTML5 Report Viewer.

Note to external contributors: make sure to sign our Contribution License Agreement (CLA) for Blazor UI first:

https://forms.office.com/Pages/ResponsePage.aspx?id=Z2om2-DLJk2uGtBYH-A1NbWxVqugKN5DvVp8I-1AgOBURFBVSkwyMlA1TkFDVFdMNU1aM1o1UlZQOC4u

closes https://github.com/telerik/blazor/issues/4911

### CODE REVIEW

**Review by jivanova — APPROVED** [review]

**Comment by dimodi** [comment]
@ighristov Thanks, Ivan! I have two questions - 

1. Is [this URL](https://docs.telerik.com/reporting/blazor-report-viewer) still valid? I see it works, but there is a redirect. Should we link to the [new one](https://docs.telerik.com/reporting/embedding-reports/display-reports-in-applications/web-application/blazor-report-viewer/overview) instead?
2. I split the long paragraph about the older Report Viewer to a bullet list, for better readability. Currently it looks like the old viewer has more features than the new one. If this is **not** true, perhaps we should tweak the content to avoid confusion?

**Comment by ighristov** [comment]
@dimodi 
Hi Dimo!
1. Yes, it's still valid - we'll keep the both Blazor Report Viewers available for some time - at least until the Native Blazor viewer is fully implemented. You are right, our latest documentation refactoring introduced this redirect, so the new URL should be used instead: https://docs.telerik.com/reporting/embedding-reports/display-reports-in-applications/web-application/blazor-report-viewer/overview
2. Correct - the old viewer has more features than the new one and this is reflected in our docs as well. Our goal is to close the features gap in the following releases and make the new viewer on par with the old one.

File: integrations/reporting.md [hunk 1]
```diff
@@ -8,15 +8,33 @@ position: 0
...
+* [Telerik Blazor Report Viewer](#telerik-blazor-report-viewer)
+* [Telerik Native Blazor Report Viewer](#telerik-native-blazor-report-viewer)
+* [Telerik Blazor Report Designer Component](#telerik-blazor-report-designer)
 
-* The [Telerik Blazor Report Designer Component](https://docs.telerik.com/reporting/how-to-blazor-web-report-designer) brings embedding of the report designer tool into your Blazor application. This empowers your trusted end-users to edit existing or create new report definitions from scratch. See the report designer into action in the [Web Report Designer Demo](https://demos.telerik.com/reporting/designer)
+## Telerik Blazor Report Viewer
+
+The [Telerik Blazor Report Viewer Component](https://docs.telerik.com/reporting/blazor-report-viewer) enables you to embed a report preview into you app. The component is built on top of our robust and reliable HTML5 Report Viewer - a JavaScript/jQuery widget with lots of customization options. See this report viewer component into action in the [Dashboard Report Demo](https://demos.telerik.com/reporting/dashboard). This report viewer includes the following functionality:
```
Comment (dimodi) [inline integrations/reporting.md:19]:
```suggestion
The [Telerik Blazor Report Viewer Component](https://docs.telerik.com/reporting/embedding-reports/display-reports-in-applications/web-application/blazor-report-viewer/overview) enables you to embed a report preview into you app. The component is built on top of our robust and reliable HTML5 Report Viewer - a JavaScript/jQuery widget with lots of customization options. See this report viewer component into action in the [Dashboard Report Demo](https://demos.telerik.com/reporting/dashboard). This report viewer includes the following functionality:
```

File: integrations/reporting.md [hunk 2]
```diff
@@ -8,15 +8,33 @@ position: 0
...
+* Use any other interactive features that the Telerik Reporting product offers.
+
+## Telerik Native Blazor Report Viewer
+
+The [Telerik Native Blazor Report Viewer Component](https://docs.telerik.com/reporting/embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/overview) is a report viewer built with **native Blazor components** from our Telerik UI for Blazor library (requires a valid license). This report viewer brings a fluent user experience that blends with the rest of your Blazor application using the same input controls and styling mechanism. The initial version of this viewer supports all the core functionality needed for previewing and exporting report documents.
+
+## Telerik Blazor Report Designer
+
+The [Telerik Blazor Report Designer Component](https://docs.telerik.com/reporting/how-to-blazor-web-report-designer) brings embedding of the report designer tool into your Blazor application. This empowers your trusted end-users to edit existing or create new report definitions from scratch. See the report designer into action in the [Web Report Designer Demo](https://demos.telerik.com/reporting/designer)
```
Comment (dimodi) [inline integrations/reporting.md:33]:
```suggestion
The [Telerik Blazor Report Designer Component](https://docs.telerik.com/reporting/how-to-blazor-web-report-designer) enables you to embed a Report Designer tool in your Blazor application. This empowers your trusted end-users to edit existing or create new report definitions from scratch. See the Report Designer into action in the [Web Report Designer Demo](https://demos.telerik.com/reporting/designer)
```

**Pull request merged**